.github/workflows/test_suite_windows.yml

```diff
@@ -57,12 +57,15 @@ jobs:
       run: python scripts/packaging/set_commit_and_date_in_version.py ${{ github.sha }}
     - name: Webots Package Creation
       run: |
-        echo $PATH
-        javac -version
         export WEBOTS_HOME=$GITHUB_WORKSPACE
         make distrib -j4
+    - name: Create/Update GitHub release
+      if: ${{ (github.event_name == 'push' || github.event_name == 'schedule') }}
+      run: |
+        python -m pip install requests PyGithub
+        scripts/packaging/publish_release.py --key=${{ secrets.GITHUB_TOKEN }} --repo=${{ github.repository }} --branch=${{ github.ref }} --commit=${{ github.sha }} --tag=${{ github.ref }}
     - uses: actions/upload-artifact@v2
-      if: ${{ contains(github.event.pull_request.labels.*.name, 'test distribution') }}
+      if: ${{ contains(github.event.pull_request.labels.*.name, 'test suite') || contains(github.event.pull_request.labels.*.name, 'test distribution') }}
```

Comment:
Why do you upload packages in case of the `test_suite` label if there is no test suite workflow for Windows?

Artifacts are only needed to pass the built packages to other jobs (like `test_suite`) but Windows.
Note that for macOS and Linux, the artifacts are deleted at the end of the workflow in case of `test_suite` label but this is currently not the case here in this version of the Windows workflow if you upload them here.

Reply:
Thank you for the review.
Let me know if it looks better now.

Reply:
Thank you.
It's fine now even if the `delete-artifacts` job will never have artifacts to remove.

Reply:
It will if we use the "test distribution" label, isn't it?

Reply:
No, because the artifacts are uploaded only if the "test distribution" label is set and deleted only if the "test distribution" label is not set.
So if the "test distribution" label is set, artifacts are not deleted.

Reply:
Oh yes... But that is ready for when we need it (when are going to introduce the nightly unit tests on Windows).